src/components/Catalogue.js

```diff
@@ -0,0 +1,5 @@
+function Catalogue() {
+  return <button type="button">Check Status</button>;
+}
+
+export default Catalogue;
```

Comment:
- [ ] (Optional) I know you are allowed to name this component whatever you like, but it would be more semantic to use the name `Categories` because it is not going to display catalogues. 👍